src/sif/pedestriancost.cc

```diff
@@ -471,7 +458,6 @@ class PedestrianCost : public DynamicCost {
     c += country_crossing_cost_ * (node->type() == baldr::NodeType::kBorderControl);
     c += gate_cost_ * (node->type() == baldr::NodeType::kGate);
     c += bike_share_cost_ * (node->type() == baldr::NodeType::kBikeShare);
-    c += toll_booth_cost_ * (node->type() == baldr::NodeType::kTollBooth);
```

Comment:
we don't parse `tool_booth_cost` option for pedestrians, so it doesn't make sense to use it here